services/connectivity/src/routes/connectivity.js

```diff
@@ -1,4 +1,5 @@
 const { Router } = require('restify-router');
+const RestifyErrors = require('restify-errors');
 
 module.exports = (errorHandler, connectivityService) => {
```

Comment:
Did you test the endpoints? I don't think this will work because you are still expecting `errorHandler` as first parameter, but it's not being sent from the dependency container

Reply:
already tested, it's crashing